assets/js/phoenix_live_view.js

```diff
@@ -1542,7 +1542,7 @@ export let DOM = {
 
   discardError(container, el, phxFeedbackFor){
     let field = el.getAttribute && el.getAttribute(phxFeedbackFor)
-    let input = field && container.querySelector(`#${field}`)
+    let input = field && container.querySelector(`#${field}, [name="${field}"]`)
```

Comment:
```suggestion
    // TODO: Remove id lookup after we updated Phoenix to use input_name instead of input_id
    let input = field && container.querySelector(`#${field}, [name="${field}"]`)
```

Reply:
@josevalim What do you think about a more specific timeline for the TODO here? After https://github.com/phoenixframework/phoenix/pull/4107 is merged, there's still going to be lots of old code that has the old error_helpers.

I think removing the id lookup needs to be part of the next minor version with specific upgrade instructions about changing the error helpers in your app.

Reply:
Or were you implying that the generator change would be part of a new phoenix version with upgrade instructions as part of that?

Reply:
@mveytsman we need to have instructions on the CHANGELOG of this project and also on Phoenix CHANGELOG. So until Phoenix v1.6 is released and a couple months have passed, we likely cannot remove this, that's why it is left intentionally vague.

Reply:
Gotcha!